site/topics/selection/selection.rst

```diff
@@ -6,6 +6,49 @@ Selection and Population Management
 Generational vs. Steady State
 =============================
 
+* All the algorithms provided so far have been implemented as a generational algorithm
+
+    * A whole new population is created based on the current population
+    * The new population becomes the current population
+    * The population size remains the same
+    * There are very discrete transitions between generations
+
+
+* With the generational algorithm, depending on how the genetic operators are applied, the new population is filled with
+
+    * Copies of unchanged selected chromosomes
+    * Mutated selected chromosomes
+    * Offspring resulting from crossover on selected chromosomes
+
+
+* On the other hand, steady state algorithms provide a more continuous type of evolution
+
+    * Not as clear discrete transitions between generations
+
+
+* With steady state
+
+    * Given a single population of size :math:`\mu`
```

Comment:
I've always wondered by population size seems to always be fixed. I feel like an analogy for steady state _should_ be biological population growth, where a portion of the population procreates, and the overall population grows. But then we freeze a couple people because they're better than everyone else, and eat the poor. It kinda falls apart at the end.

Reply:
The generational is just what people tend to do these days as it works and it's easier. 

In reality, like everything with EAs, it's _idk, do whatever you want, that's the fun of this stuff_. You can find all these variations of people having populations that just get bigger and bigger, some get bigger and bigger and then have extinction events, some have ways to grow and shrink. It's chaos really.